ydb/library/yql/minikql/comp_nodes/mkql_grace_join.cpp

```diff
@@ -767,7 +768,9 @@ class TGraceJoinSpillingSupportState : public TComputationValue<TGraceJoinSpilli
                 const auto used = TlsAllocState->GetUsed();
                 const auto limit = TlsAllocState->GetLimit();
 
-                YQL_LOG(INFO) << "yellow zone reached " << (used*100/limit) << "%=" << used << "/" << limit;
+                if (limit) {
```

Comment:
Please move this log to a separate function and use different log message in case of limit = 0. Use function like this:
https://github.com/ydb-platform/ydb/blob/main/ydb/library/yql/minikql/comp_nodes/mkql_wide_combine.cpp#L549

Reply:
I've fixed that, but I'm curious why it should be in separate function.

Reply:
The main point is not to move it to a separate function. But to make the code similar to the existing place in WideCombiner. Well, and to have a meaningful message at 0 limit (although such a situation is possible only in tests)